Unmet dependency for Windows client

```
0.12s$ GOOS=windows go build -o duckdns-windows
../../sirupsen/logrus/terminal_windows.go:10:2: cannot find package "github.com/konsorten/go-windows-terminal-sequences" in any of:
	/home/travis/.gimme/versions/go1.8.linux.amd64/src/github.com/konsorten/go-windows-terminal-sequences (from $GOROOT)
	/home/travis/gopath/src/github.com/konsorten/go-windows-terminal-sequences (from $GOPATH)
The command "GOOS=windows go build -o duckdns-windows" exited with 1.
Done. Your build exited with 1.
```

This occurs for all versions of Go. The `logrus` library has been a source of pain since the lower-casing of the repo owner earlier this year. If `vgo` or `dep` solve this more gracefully than `glide` I'm up for that. There's potentially a fix in the `.travis.yml` to be had as the dependency management line is... not robust.

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,19 +1,26 @@
 language: go
 sudo: false
 
+go:
+  - "1.8.x"
+  - "1.9.x"
+  - "1.10.x"
+  - "1.11.x"
+  - tip
+env:
+  - GIMME_OS=linux
+  - GIMME_OS=darwin
+  - GIMME_OS=windows
+
 matrix:
-  include:
-    - go: 1.8
-    - go: 1.9
+  allow_failures:
     - go: tip
+  fast_finish: true
 
 install:
-  - # Skip
+  - go get -d -v ./...
 
 script:
-  - go get -t -v ./...
   - diff -u <(echo -n) <(gofmt -d .)
   - go tool vet .
-  - GOOS=linux go build -o duckdns-linux
-  - GOOS=darwin go build -o duckdns-macos
-  - GOOS=windows go build -o duckdns-windows
+  - go build -v ./...

